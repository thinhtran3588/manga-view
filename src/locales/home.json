{
  "EN": {
    "SEARCH_MANGA": "Search manga",
    "SEARCHING": "Searching",
    "LOADING": "Loading",
    "RECENT": "Recent"
  },
  "VI": {
    "SEARCH_MANGA": "Tìm truyện",
    "SEARCHING": "Đang tìm kiếm",
    "LOADING": "Đang tải",
    "RECENT": "Gần đây"
  }
}
