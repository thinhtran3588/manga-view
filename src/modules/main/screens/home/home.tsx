import type {NextPage} from 'next';
import {Seo} from '@core/components/seo';

export const Home: NextPage = (): JSX.Element => (
  <>
    <Seo />
    <div className=''>demo</div>
  </>
);
