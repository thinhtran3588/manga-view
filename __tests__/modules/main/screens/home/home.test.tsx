/**
 * @jest-environment jsdom
 */

import React from 'react';
import {render} from '@test/test-utils';
import {Home} from '@main/screens/home/home';

describe('Home', () => {
  it('renders successfully', async () => {
    const {container} = render(<Home />);

    expect(container).toMatchInlineSnapshot(`
<div>
  <div
    class="m-2"
  >
    demo
    <button
      class="bg-gradient dark:bg-gradient-light py-2 px-4 rounded-full text-white font-bold"
    >
      Test button
    </button>
  </div>
</div>
`);
  });
});
