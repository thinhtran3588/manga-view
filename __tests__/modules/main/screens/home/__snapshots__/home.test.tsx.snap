// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders successfully 1`] = `
<div>
  <div
    class="flex flex-col w-full"
  >
    <div
      class="flex rounded-full bg-white dark:bg-gray-600 p-2 shadow-xl w-full mb-2"
    >
      <svg
        class="h-6 w-6"
        fill="none"
        stroke="currentColor"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
        />
      </svg>
      <input
        class="bg-transparent outline-none ml-2 flex-1"
        placeholder="Search manga"
        value=""
      />
      
    </div>
    
  </div>
</div>
`;
